common/network-common/pom.xml

```diff
@@ -45,6 +45,22 @@
       <artifactId>commons-lang3</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.fusesource.leveldbjni</groupId>
```

Comment:
@tgravescs Why not move LevelDBProvider and this dependency to `common/network-shuffle/`? It's not used in `common/network-common/`.